iocore/net/UnixNetVConnection.cc

```diff
@@ -1273,14 +1273,21 @@ UnixNetVConnection::connectUp(EThread *t, int fd)
     con.is_bound     = true;
   }
 
+  if (check_emergency_throttle(con)) {
```

Comment:
changed the return type to bool for emergency_throttle, check_net_throttle and check_emergency_throttle